release-1.6/release_team.md

```diff
@@ -0,0 +1,12 @@
+|  **Role** | **Name** |
+|  ------ | ------ |
+|  Lead | Dan Gillespie |
+|  **Secondary** |  |
+|  Features | Ihor Dvoretskyi |
+|  Release Branch | Anthony Yeh |
+|  **Infra** | Jeff Grafton |
+|  Docs | Devin Donnelly |
+|  Bugs | Caleb Miles |
+|  **Testing** | Maru Newby |
```

Comment:
@spiffxp I've been using this [spreadsheet](https://docs.google.com/spreadsheets/d/1OvBDDzVeqzktyGZSD4-eMfnJgJros-86rgHQrrYjP-s/edit#gid=0) as a source, created by @ethernetdan - no objections to modify the font.